### PR TITLE
Arrival data entry updates and other misc. changes

### LINKED ISSUE #294: [NIRC] Basic Colony Management - Arrival form and Misc. updates and fixes
### Tasks
- [x] Arrival - update 'source' column in ETL
- [x] Add My Tasks link on main page
- [x] Arrival form - add housing record
- [x] Hide QCState on Death/Necropsy form
- [x] Fix NIRC_EHRTest.testCalculatedAgeColumns
- [x] Fix NIRC_EHRTest.testDeathNecropsyForm
- [x] Arrival/Departure - Add source/destination lookup
- [x] Add geographic origin in Arrival form
- [x] Code review (s) - @ankurjuneja @labkey-martyp 
- [x] Teamcity review and merge -1

### PR DESCRIPTION
**Pull request by labkey-bpatel**
#### Rationale
Fixes and updates on Arrival form, and other misc. updates mentioned below.

#### Related Pull Requests
* 

#### Changes
* Update 'source' for Arrival ETL.
* Add housing record from Arrival data entry. 
* Fix Arrival trigger to be able to update demographics, birth, housing, and protocol and project assignments. 
* Hide QC State from Death/Necropsy form.
* Add source for Arrival and Departure data entry forms.
* Add geographic origin.
* Sample data updates, test fix.
* Add "My Review Tasks" link on the main page.
* Add geographic_origin to Arrivals form.
* Trim 'performedBy' in ETLs

### CODE REVIEW

File: nirc_ehr/resources/views/begin.html [hunk 1]
```diff
@@ -26,15 +26,62 @@
...
+                        href: item.url ? item.url : item.urlConfig ? LABKEY.ActionURL.buildURL(item.urlConfig.controller, item.urlConfig.action, null, item.urlConfig.params) : null,
+                        tooltip: item.tooltip,
+                        showBrackets: false
+                    }]
+                }
+            }
         });
         const renderPage = () => {
             let issueHTML = '';
+            console.log("this.hasMyTasks = "+ this.myNumOfTasks);
```
Comment (ankurjuneja) [inline nirc_ehr/resources/views/begin.html]: is console.log needed?

File: nirc_ehr/resources/web/nirc_ehr/model/sources/DeathNecropsy.js [hunk 2]
```diff
@@ -23,6 +23,12 @@ EHR.model.DataModelManager.registerMetadata('DeathNecropsy', {
         }
     },
     byQuery: {
+        'study.deaths': {
+            qcstate:
+            {
```
Comment (ankurjuneja) [inline nirc_ehr/resources/web/nirc_ehr/model/sources/DeathNecropsy.js]:
```suggestion
            qcstate: {
```

File: nirc_ehr/resources/web/nirc_ehr/model/sources/DeathNecropsy.js [hunk 3]
```diff
@@ -91,6 +97,10 @@ EHR.model.DataModelManager.registerMetadata('DeathNecropsy', {
             category: {
                 hidden: true
             },
+            qcstate:
+            {
```
Comment (ankurjuneja) [inline nirc_ehr/resources/web/nirc_ehr/model/sources/DeathNecropsy.js]:
```suggestion
            qcstate: {
```

File: nirc_ehr/test/src/org.labkey.test/tests.nirc_ehr/NIRC_EHRTest.java [hunk 4]
```diff
@@ -168,8 +168,8 @@ protected void populateInitialData() throws Exception
     @BeforeClass
     public static void setupProject() throws Exception
     {
-        NIRC_EHRTest init = (NIRC_EHRTest) getCurrentTest();
-        init.doSetup();
+//        NIRC_EHRTest init = (NIRC_EHRTest) getCurrentTest();
+//        init.doSetup();
```
Comment (ankurjuneja) [inline nirc_ehr/test/src/org.labkey.test/tests.nirc_ehr/NIRC_EHRTest.java]: I think this is not intentional.

**Review by ankurjuneja — CHANGES_REQUESTED** [review]
One small request in NIRC_EHRTest to undo commenting out the test setup, rest are suggestions.

File: nirc_ehr/resources/views/begin.html (hunk 1 above)
Comment (labkey-bpatel) [reply nirc_ehr/resources/views/begin.html]: nope, removed

File: nirc_ehr/test/src/org.labkey.test/tests.nirc_ehr/NIRC_EHRTest.java (hunk 4 above)
Comment (labkey-bpatel) [reply nirc_ehr/test/src/org.labkey.test/tests.nirc_ehr/NIRC_EHRTest.java]: good catch, now uncommented!

**Review by ankurjuneja — APPROVED** [review]

**Pull request merged**